eng/testing/tests.mobile.targets

```diff
@@ -71,108 +79,6 @@
              AssemblyFile="$(MonoTargetsTasksAssemblyPath)"
              Condition="'$(MonoTargetsTasksAssemblyPath)' != ''" />
 
-  <Target Name="GenerateRuntimeConfig" Condition="'$(TargetOS)' != 'Browser'"
-                                       Inputs="$(PublishDir)$(AssemblyName).runtimeconfig.json"
-                                       Outputs="$(PublishDir)runtimeconfig.bin">
-    <PropertyGroup>
-      <RuntimeConfigFilePath>$(PublishDir)$(AssemblyName).runtimeconfig.json</RuntimeConfigFilePath>
-      <ParsedRuntimeConfigFilePath>$(PublishDir)runtimeconfig.bin</ParsedRuntimeConfigFilePath>
-    </PropertyGroup>
-
-    <ItemGroup>
-      <RuntimeConfigReservedProperties Include="RUNTIME_IDENTIFIER"/>
-      <RuntimeConfigReservedProperties Include="APP_CONTEXT_BASE_DIRECTORY"/>
-    </ItemGroup>
-
-    <!-- Parse runtimeconfig.template.json file -->
-    <RuntimeConfigParserTask
-        RuntimeConfigFile="$(RuntimeConfigFilePath)"
-        OutputFile="$(ParsedRuntimeConfigFilePath)"
-        RuntimeConfigReservedProperties="@(RuntimeConfigReservedProperties)">
-    </RuntimeConfigParserTask>
-  </Target>
-
-  <!-- Generate a self-contained app bundle for Android with tests. -->
-  <UsingTask Condition="'$(TargetOS)' == 'Android'"
-             TaskName="AndroidAppBuilderTask"
-             AssemblyFile="$(AndroidAppBuilderTasksAssemblyPath)" />
-
-  <Target Condition="'$(TargetOS)' == 'Android'" Name="BundleTestAndroidApp" DependsOnTargets="GenerateRuntimeConfig">
```

Comment:
Need to remove L11 that declares `<BundleTestAppTargets>BundleTestAndroidApp</BundleTestAppTargets>` as well. might fix the build problems?